packages/playground/src/components/execution/key-value-table.tsx

```diff
@@ -0,0 +1,118 @@
+import React from 'react'
+import {
+  Table,
+  TableBody,
+  TableCell,
+  TableHeader,
+  TableHead,
+  TableRow,
+  Text,
+  Accordion,
+  AccordionContent,
+  AccordionItem,
+  AccordionTrigger
+} from '@harnessio/canary'
+
+type KeyValuePair = {
+  name: string
+  value: string | KeyValuePair[]
+}
+
+interface KeyValueTableProps {
+  tableTitleName: string
+  tableTitleVal: string
+  className?: string
+  tableVal: KeyValuePair
```

Comment:
Shouldn't it be `tableVal: KeyValuePair[]`?

Reply:
Yes, I feel like it's too similar with tableTitleVal. I will change the name to tableSpec: KeyValuePair[] to make easier to identify what it is.